ethan.py

```diff
@@ -0,0 +1 @@
+print('Hello World! This is my contribution to the test-hello-2020 repo')
```

Comment:
Convention is to finalise your .py files with a bare newline, could you add that? 😄